docs/migration/migrate_7_0.asciidoc

```diff
@@ -119,6 +119,30 @@ Previously set `xpack.monitoring.elasticsearch.ssl.verify` is equal to `xpack.mo
 
 *Impact:* Users with `xpack.monitoring.report_stats` set should use `xpack.xpack_main.telemetry.enabled` instead.
 
+[float]
+=== kibana.yml setting `xpack.monitoring.index_pattern` is no longer valid
+*Details:* This unsupported setting has been officially removed. The ability to customize this pattern is no longer supported.
+
+[float]
+=== kibana.yml setting `xpack.monitoring.kibana.index_pattern` is no longer valid
+*Details:* This unsupported setting has been officially removed. The ability to customize this pattern is no longer supported
```

Comment:
From this line on, the first sentence in each block ends with a period but the second does not. :)